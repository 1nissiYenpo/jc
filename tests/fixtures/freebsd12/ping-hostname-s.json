{"destination_ip": "127.0.0.1", "data_bytes": 40, "pattern": null, "destination": "localhost", "packets_transmitted": 3, "packets_received": 3, "packet_loss_percent": 0.0, "round_trip_ms_min": 0.022, "round_trip_ms_avg": 0.03, "round_trip_ms_max": 0.038, "round_trip_ms_stddev": 0.006, "responses": [{"type": "reply", "bytes": 48, "response_ip": "127.0.0.1", "icmp_seq": 0, "ttl": 64, "time_ms": 0.022}, {"type": "reply", "bytes": 48, "response_ip": "127.0.0.1", "icmp_seq": 1, "ttl": 64, "time_ms": 0.038}, {"type": "reply", "bytes": 48, "response_ip": "127.0.0.1", "icmp_seq": 2, "ttl": 64, "time_ms": 0.03}]}
