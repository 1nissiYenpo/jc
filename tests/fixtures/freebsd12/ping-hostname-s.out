PING localhost (127.0.0.1): 40 data bytes
48 bytes from 127.0.0.1: icmp_seq=0 ttl=64 time=0.022 ms
48 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.038 ms
48 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.030 ms

--- localhost ping statistics ---
3 packets transmitted, 3 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.022/0.030/0.038/0.006 ms
